src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -66,6 +68,7 @@ public function __construct(array $defaultOptions = [], int $maxHostConnections
         }
 
         $this->multi = $multi = new CurlClientState();
+        self::$curlVersion = self::$curlVersion ?? curl_version();
```

Comment:
Calling `curl_version()` repeatidily is leaking memory - I suppose the array created by the extension is not collected as userland arrays.

Reply:
I don't think that's a bug - token_get_all() has the same "leak", reclaimed when calling https://www.php.net/manual/en/function.gc-mem-caches.php